app/controllers/books_controller.rb

```diff
@@ -0,0 +1,12 @@
+class BooksController < ApplicationController
+
+  def show
+    @book = Book.find(params[:id])
+  end
+
+  private
+
+  def book_params
+    params.require(:book).permit(:title)
+  end
```

Comment:
Since you're not creating or updating you don't need a strong params. Those are only used to check parameters coming in from forms and used when you would be mutating your data.